### PR TITLE
AFF object RONT

### PR DESCRIPTION
**Pull request by rauva**
See #5852 
https://github.com/abapGit-tests/RONT.git

### CODE REVIEW

File: src/objects/aff/zcl_abapgit_object_ront.clas.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <VSEOCLASS>
+    <CLSNAME>ZCL_ABAPGIT_OBJECT_RONT</CLSNAME>
+    <LANGU>E</LANGU>
+    <DESCRIPT>ABAPGIT Objecthandler for EVBT</DESCRIPT>
```
Comment (mbtools) [inline src/objects/aff/zcl_abapgit_object_ront.clas.xml:8]: The class description should be `RONT - SAP Object Type` (see other `zcl_abapgit_object_*` classes)

**Review by mbtools — COMMENTED** [review]
I updated the test repo. Looks ok but can't test it.

What will be the minimum SAP Basis release for RONT?

**Comment by rauva** [comment]
RONT and NONT will be available with 7.58 onPrem.

**Comment by rauva** [comment]
I think it's ready to merge.

**Pull request merged**